src/webextension/list/components/item-list.css

```diff
@@ -5,25 +5,22 @@
 .empty {
   text-align: center;
   white-space: pre-wrap;
-  margin: 1em 1em;
-  padding: 20px;
+  padding: 40px;
   overflow: hidden;
   overflow-wrap: break-word;
   color: #737373;
   font-size: 15px;
   line-height: 1.5;
   letter-spacing: 0.2px;
+  border-top: 1px solid #d7d7db;
```

Comment:
Let's remove this. Based on the spec, we'll have to remove it when we redo the search box anyway.